bin/check_website-copy_paths.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e # Make script exit when a command fail.
+set -u # Exit on usage of undeclared variable.
+# set -x # Trace what gets executed.
+set -o pipefail # Catch failures in pipes.
+
+# Command line arguments
+WEBSITE_COPY_DIR=$1
+
+# Extract comment paths
+COMMENT_PATHS=$(echo -n "--extract-comment-paths" | node ./bin/cli.js)
```

Comment:
Presumably there is a reason for this slgihtly strange syntax that I don't understand?

Reply:
There is...
`cli.js` is written to take an input from stdin (JSON output from elm-review) and I didn't feel like making a different interface so...
1. I used `echo ... | node ./bin/cli.js` to pipe a single string "--extract-comment-paths" but...
2. `echo` adds a newline so I used `-n` that suppresses it

The output saved in `COMMENT_PATHS` is
```
analyzer-comments/elm/feedback_request.md
analyzer-comments/elm/common/no_unused/custom_type_constructors.md
analyzer-comments/elm/common/no_unused/custom_type_constructor_args.md
analyzer-comments/elm/common/simplify.md
analyzer-comments/elm/bettys-bike-shop/use_signature.md
analyzer-comments/elm/bettys-bike-shop/import_string.md
...
```